definitions/npm/react-jss_v6.1.x/flow_v0.30.x-/react-jss_v6.1.x.js

```diff
@@ -8,6 +8,8 @@ declare module 'react-jss' {
 
   declare type Null = null | void;
 
+  declare type Dictionary = { [key: string]: {} }
```

Comment:
same like jss.createStyleSheet + first argument can be a function that receives a theme object